scripts/view_splitter.js

```diff
@@ -35,7 +34,8 @@ var viewSplitter = function(container, storageKey) {
     });
 
     function setSplitControls() {
-        splitImage.src = splitVertical ? proofIntData.buttonImages.imgHSplit : proofIntData.buttonImages.imgVSplit;
+        buttonContent.className = splitVertical ? 'fas fa-grip-lines' : 'fas fa-grip-lines-vertical';
```

Comment:
I wonder if we could use `fal fa-columns` for the vertical and use CSS to rotate it 90 degrees for the horizontal?

https://fontawesome.com/v5/icons/columns?f=classic&s=light

Reply:
That would look better. I'll try it.

Reply:
Bummer the `fal` is in the pro version. Can we try the `fa-columns` and see how it looks in a sandbox? `fa-grip-lines` doesn't say "columns" & "rows" to me and I worry it won't be intuitive to some users.

Reply:
Implemented this now.